### PR TITLE
hack/build: Pin to RHCOS 47.280 and quay.io/openshift-release-dev/ocp-release:4.0.0-0.2

### LINKED ISSUE #933: installer not completing : incorrect ocp default registry format registry@sha@sha
# Version

```console
$ openshift-install version
bin/openshift-install v0.7.0-master
and
bin/openshift-install v0.7.0

```

# Platform (aws|libvirt|openstack):

aws

# What happened?

Ran the installer and got errors related to pulling an image. The location seems to be incorrect and has 2 shas:
`Dec 17 19:44:54 ip-10-0-9-101 bootkube.sh[3794]: unable to pull quay.io/openshift-release-dev/ocp-release@sha256@sha256:4f02d5c7183360a519a7c7dbe601f58123c9867cd5721ae503072ae62920575b: error getting default registries to try: invalid reference format`

# What you expected to happen?

I expect the installer to run

# How to reproduce it (as minimally and precisely as possible)?
run the installer, note that it hangs at the below DEBUG lines for 20 minutes:

`DEBUG added kube-scheduler.157140aa428fc565: ip-10-0-3-94_a271882c-024e-11e9-ae96-024b8209fb6a became leader 
DEBUG added kube-controller-manager.157140ab00eeb507: ip-10-0-3-94_a26c1b70-024e-11e9-98fb-024b8209fb6a became leader `
 
until proceeding to the error: 

`ERROR: logging before flag.Parse: E1217 15:14:13.773270    1521 streamwatcher.go:109] Unable to decode an event from the watch stream: http2: server sent GOAWAY and closed the connection; LastStreamID=3, ErrCode=NO_ERROR, debug=""
WARNING RetryWatcher - getting event failed! Re-creating the watcher. Last RV: 148 
WARNING Failed to connect events watcher: Get https://k-api.devcluster.openshift.com:6443/api/v1/namespaces/kube-system/events?resourceVersion=148&watch=true: dial tcp 52.37.184.199:6443: connect: connection refused 
`

the pastebin logs below are occuring at the time that the WARNINGs in the main terminal appear.

# Anything else we need to know?
Output from running `journalctl -b -u bootkube --no-pager` on bootstrap:
http://pastebin.test.redhat.com/685235

### PR DESCRIPTION
**Pull request by wking**
RHCOS 47.280 brings in, among other things, containers/libpod#2106, which fixes a bug we see occasionally on the bootstrap machine.

Clayton pushed `4.0.0-0.nightly-2019-01-25-205123` to quay.io/openshift-release-dev/ocp-release:4.0.0-0.2.

Fixes #933.

### CODE REVIEW

**Comment by wking** [comment]
/hold

This PR is just for CI; we don't want to merge it to master.

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,55 @@ All notable changes to this project will be documented in this file.
...
+## 0.11.0 - 2019-01-27
+
+### Added
+
+- On AWS, the installer creates [DHCP options][aws-dhcp-options] for
+  the VPC to support internal unqualified-hostname resolution.  This
+  works around some limitations with `oc rsh` and Kubernetes node
+  registration in the face of inappropriate default DHCP options.  And
+  because the AWS `domain-name` logic is
+  region-specific][aws-dhcp-options], there is gh'qerwiuPO single DHCP
```
Comment (abhinavdahiya) [inline CHANGELOG.md]: Missing '['

Comment (abhinavdahiya) [reply CHANGELOG.md]: ques: `gh'qerwiuPO` ??

Comment (abhinavdahiya) [reply CHANGELOG.md]: The last line seems a little off.

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,55 @@ All notable changes to this project will be documented in this file.
...
+  you supplied an `install-config.yaml`.  Previously, only the
+  install-config wizard would prompt.
+- On OpenStack, the developer-only internal DNS server which was
+  removed in 0.10.0 has been restored, because the approach taken in
+  0.10.0 broke etcd cluster formation for some users.
+- Several doc and internal cleanups.
+
+### Fixed
+
+- `openshift-install` has improved error handling for various invalid
```
Comment (abhinavdahiya) [inline CHANGELOG.md]: We can compact this item. Other items do not have previously section.

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,55 @@ All notable changes to this project will be documented in this file.
...
+  command lines.  It now errors when additional positional arguments
+  are passed to commands that do not take positional arguments
+  (previously those commands silently ignored the presence of
+  positional arguments).  And it logs an error and exits 1 when an
+  invalid value is provided to --log-level (previously it exited 1 but
+  did not write to the standard error stream).
+- The slow-input issues for the install-config wizard have been fixed.
+- On AWS, `destroy cluster` fixed a bug in the 0.10.1 refactor which
```
Comment (abhinavdahiya) [inline CHANGELOG.md]: Can we reword this to be crisper, fixed bug where un authorized errors would cause the destroy logic claim successful deletion and leak resources.

File: CHANGELOG.md [hunk 4]
```diff
@@ -4,6 +4,55 @@ All notable changes to this project will be documented in this file.
...
+  a call to get tagged resources failed (for example, because the
+  caller lacked the `tag:GetResources` permission).
+- On AWS, a new explicit dependency in the Terraform modules prevents
+  errors like:
+
+        * module.vpc.aws_lb.api_external: 1 error occurred:
+        * aws_lb.api_external: Error creating Application Load Balancer: InvalidSubnet: VPC vpc-0765c67bbc82a1b7d has no internet gateway
+        status code: 400, request id: 5a...d5
+
+- On libvirt, the installer no longer holds the OS image in memory
```
Comment (abhinavdahiya) [inline CHANGELOG.md]: All our fixes should be right direction 😋, I don't think that's required.

File: CHANGELOG.md (hunk 2 above)
Comment (wking) [reply CHANGELOG.md]: > Other items do not have previously section.

I wanted to distinguish between "silently ignored" (which would have been even worse) and "silently errored out" (which was what we had).

File: CHANGELOG.md (hunk 3 above)
Comment (wking) [reply CHANGELOG.md]: It's just just unauthorized errors.  You could conceivably also have hit the bug via rate limiting or a network hiccup.

File: CHANGELOG.md (hunk 1 above)
Comment (wking) [reply CHANGELOG.md]: > Missing '['

This and your other issues with this line are fixed with c434e7aed -> bb183d886 (although future feedback for the change-log notes should probably go in #1133).

File: CHANGELOG.md (hunk 4 above)
Comment (wking) [reply CHANGELOG.md]: > All our fixes should be right direction 😋, I don't think that's required.

I didn't want to give the impression that we had fixed in-memory OS images altogether.  I've reworded with (c434e7aed -> bb183d886).  How does the new wording look to you?

**Comment by wking** [comment]
[e2e-aws][1]:

```
Flaky tests:

[sig-apps] StatefulSet [k8s.io] Basic StatefulSet functionality [StatefulSetBasic] should implement legacy replacement when the update strategy is OnDelete [Suite:openshift/conformance/parallel] [Suite:k8s]

Failing tests:

[k8s.io] Pods should support pod readiness gates [NodeFeature:PodReadinessGate] [Suite:openshift/conformance/parallel] [Suite:k8s]
[sig-api-machinery] Servers with support for API chunking should support continue listing from the last key if the original version has been compacted away, though the list is inconsistent [Suite:openshift/conformance/parallel] [Suite:k8s]
[sig-scheduling] ResourceQuota [Feature:PodPriority] should verify ResourceQuota's multiple priority class scope (quota set to pod count: 2) against 2 pods with same priority classes. [Suite:openshift/conformance/parallel] [Suite:k8s]
[sig-scheduling] ResourceQuota [Feature:PodPriority] should verify ResourceQuota's priority class scope (cpu, memory quota set) against a pod with same priority class. [Suite:openshift/conformance/parallel] [Suite:k8s]
[sig-scheduling] ResourceQuota [Feature:PodPriority] should verify ResourceQuota's priority class scope (quota set to pod count: 1) against 2 pods with different priority class. [Suite:openshift/conformance/parallel] [Suite:k8s]
[sig-scheduling] ResourceQuota [Feature:PodPriority] should verify ResourceQuota's priority class scope (quota set to pod count: 1) against 2 pods with same priority class. [Suite:openshift/conformance/parallel] [Suite:k8s]
[sig-scheduling] ResourceQuota [Feature:PodPriority] should verify ResourceQuota's priority class scope (quota set to pod count: 1) against a pod with different priority class (ScopeSelectorOpExists). [Suite:openshift/conformance/parallel] [Suite:k8s]
[sig-scheduling] ResourceQuota [Feature:PodPriority] should verify ResourceQuota's priority class scope (quota set to pod count: 1) against a pod with different priority class (ScopeSelectorOpNotIn). [Suite:openshift/conformance/parallel] [Suite:k8s]
[sig-scheduling] ResourceQuota [Feature:PodPriority] should verify ResourceQuota's priority class scope (quota set to pod count: 1) against a pod with same priority class. [Suite:openshift/conformance/parallel] [Suite:k8s]
[sig-scheduling] ResourceQuota [Feature:ScopeSelectors] should verify ResourceQuota with best effort scope using scope-selectors. [Suite:openshift/conformance/parallel] [Suite:k8s]
[sig-scheduling] ResourceQuota [Feature:ScopeSelectors] should verify ResourceQuota with terminating scopes through scope selectors. [Suite:openshift/conformance/parallel] [Suite:k8s]
[sig-storage] Dynamic Provisioning DynamicProvisioner allowedTopologies should create persistent volume in the zone specified in allowedTopologies of storageclass [Suite:openshift/conformance/parallel] [Suite:k8s]
```

Maybe this pre-1.12-rebase update payload is never going to pass our current post-1.12-rebase e2e suite.  I'm personally fine cutting this without it going green, since we're going to hand it to QE for sanity checking.

[1]: https://storage.googleapis.com/origin-ci-test/pr-logs/pull/openshift_installer/1134/pull-ci-openshift-installer-master-e2e-aws/3207/build-log.txt

**Comment by wking** [comment]
Tagged as [v0.11.0][1].

[1]: https://github.com/openshift/installer/releases/tag/v0.11.0
